src/Storage/PathBuilder/BasePathBuilder.php

```diff
@@ -214,6 +215,22 @@ protected function _buildFilename(EntityInterface $entity, array $options = [])
 	}
 
 /**
+ * Checks if a subdirectory has been set in the adapter settings.
+ *
+ * Used to add to the URL functions.
+ *
+ * @param string $adapter Adapter config name set in the StorageManager
+ * @return string
+ */
+	protected function _adapterSubDirectory($adapter) {
```

Comment:
ah no! my bad, I thought it was a generic option for all the adapters but I just had a look and its not.  Strange why they would add it only for the S3 adapter.

Either way I think its a silly option.